tests/cases/classes/method-type-case

```diff
@@ -11,10 +11,16 @@ namespace Vendor;
 
 abstract class Foo
 {
+    /**
+     * @param array<string> $qux
```

Comment:
This will depend on https://github.com/slevomat/coding-standard/pull/558. (PHPStan already supports it, next version of PHPStorm will.)

Reply:
Options to not block this:

1. Don't include the rule.
2. Use 'traditional' syntax (eg `string[]`). Means having to change them all in the future, but means can benefit from being clearer now (and will hopefully be auto-fixable).

Reply:
Though with 2 means can't use generics at all, which creates a problem for the likes of https://github.com/libero/content-api-bundle/blob/475e0b29a01b8e056eb549fb39378c55a246e7d6/src/Adapter/InMemoryItems.php#L27.

Going to suggest we do 1.